src/k8s/pkg/k8sd/features/cilium/internal.go

```diff
@@ -0,0 +1,34 @@
+package cilium
+
+import (
+	"github.com/canonical/k8s/pkg/k8sd/types"
+)
+
+const (
+	// List of devices facing cluster/external network (used for BPF NodePort, BPF masquerading and host firewall); supports '+' as wildcard in device name, e.g. 'eth+'
+	// e.g. k8sd/v1alpha1/cilium/devices="eth+ lxdbr+"
+	annotationDevices = "k8sd/v1alpha1/cilium/devices"
```

Comment:
Those should be part of the API https://github.com/canonical/k8s-snap-api/blob/main/api/v1/const.go

Reply:
Please directly add them also to the docs page (IMHO this two lines do not justify a new card).

Reply:
I'm not sure about this, we don't have calico related bits in the api as well. Should we put feature dependent annotations on the api directly as these can be interchanged?

Reply:
I think it's a good idea to move all these annotations to the API repo. Maybe in the future we can also move Calico's (or any other feature) annotations there as well.

Reply:
If the annotations are moved there, we might also consider re-organizing them from being only in the `consts.go` file to a more structured format that better conveys the annotations intention (something for the future ofc).